src/pretix/helpers/enums.py

```diff
@@ -0,0 +1,6 @@
+from enum import StrEnum
+
+
+class ValidStates(StrEnum):
```

Comment:
Please move this definition to `pretix/common/`. We reserve this module for basic data types, which can be imported by other apps without causing side effect for circular import error.